docs/overlay.md

```diff
@@ -1,6 +1,154 @@
-Overlay Driver
-==============
+# Overlay Driver
 
-## Configuration
+### Multi-Host Overlay Driver Quick Start
 
-## Usage
+This example is to provision two Docker Hosts with the **experimental** Libnetwork default overlay network driver.
+
+### Pre-Requisites
+
+- Kernel >= 3.1.6
+- Install Docker Experimental
```

Comment:
I dont think installing docker experimental is a prereq. Its part of the procedure that you have listed down